system-addon/content-src/components/ComponentPerfTimer/ComponentPerfTimer.jsx

```diff
@@ -103,6 +103,7 @@ class ComponentPerfTimer extends React.Component {
    * Creates `TELEMETRY_UNDESIRED_EVENT` with timestamp in ms
    * of how much longer the data took to be ready for display than it would
    * have been the ideal case.
+   * https://github.com/mozilla/ping-centre/issues/98
```

Comment:
So that issue appears to be where they ensured that the tables are available to receive the data.  The actual user story was originally at mozilla/activity-stream#2841 and ultimately moved to mozilla/ping-centre#88.  If it's worth linking to anything from the code, (I have mixed feelings), it's probably that last one.